docs/release-pex-models.md

```diff
@@ -49,3 +49,13 @@ yarn publish --access public
 Check on `npmjs.com`
 
 Install the module following the included `README.md`. 
+
+
+## Releasing java
+
+If you have the correct settings in your local workspace, you can easily publish a new version on Sphereon repository (`sphereon-opensource-snapshots`)
```

Comment:
What is the correct settings for the local workspace? Can you also add some detail for this?

Reply:
I'm not sure what I should put here, you have to have correct settings.xml in your maven home to be able to connect, you have to have the correct credentials

Reply:
Yeah agree. Let's not explain to us internally how you use maven